### PR TITLE
feat: add workflows PR & master

### PR DESCRIPTION
**Pull request by Abdel-Monaam-Aouini**

### CODE REVIEW

File: lib/rules/rules.json [hunk 1]
```diff
@@ -47,8 +47,8 @@
         "ES6": true
     },
     "first": {
-        "compatible": true,
-        "alternative": "Array.prototype.slice() or arr[0]"
+      "compatible": true,
+      "alternative": "Array.prototype.slice()"
```
Comment (ODudek) [inline lib/rules/rules.json:51]: Why did you remove `or arr[0]` part?

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        run: gh pr review --approve "$PR_URL"
+        env:
+          PR_URL: ${{github.event.pull_request.html_url}}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Enable auto-merge for Dependabot PRs
+        if: ${{ steps.dependabot-metadata.outputs.update-type != 'version-update:semver-major' }}
+        run: gh pr merge --auto --squash "$PR_URL"
+        env:
+          PR_URL: ${{ github.event.pull_request.html_url }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (ODudek) [inline .github/workflows/pr.yml:59]: add new line at the end of file

File: .github/workflows/master.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    environment: testing
+    steps:
+      - name: 👌🏻 Checkout
+        uses: actions/checkout@v3
+      - name: Install dependencies
+        run: npm ci
+      - name: 🔎 Run tests
+        run: npm run test
+      - name: Run unit tests
+        run: npm run test:unit
```
Comment (ODudek) [inline .github/workflows/master.yml:29]: same here, new line

**Review by ODudek — CHANGES_REQUESTED** [review]

File: lib/rules/rules.json (hunk 1 above)
Comment (Abdel-Monaam-Aouini) [reply lib/rules/rules.json:51]: Please run `test:unit` and you'll see this error :)

**Comment by Abdel-Monaam-Aouini** [comment]
@ODudek any update here ?

**Comment by ODudek** [comment]
`npm ci` can only be used if the repository contains a `package-lock.json` file. Could you please add it in the new PR, @Abdel-Monaam-Aouini ?
